distributions/validate.py

```diff
@@ -70,8 +70,15 @@ def is_unique(collection: list):
     distros = content['Distributions']
     assert is_unique([e.get('StoreAppId') for e in distros if e])
     assert is_unique([e.get('Name') for e in distros if e])
+    
+    specificDistro = ""
+
+    if len(sys.argv) > 2:
+        specificDistro = sys.argv[2]
```

Comment:
nit: We could take a different branch and do: 
```
    if len(sys.argv) > 2:
        content =[e for e in content if e['name'] == sys.argv[2]]
        if not content:
            raise RuntimeError(f'No distribution found for name: {sys.argv[2]}')
```

So that way the script fails if we pass an invalid distribution name

Reply:
Fixed!